### PR TITLE
Add pacman extension

### PR DESCRIPTION
**Pull request by benedictdudel**
Extension for the [pacman package manager](https://wiki.archlinux.org/index.php/pacman). With this extension you can search for packages and have the options to install, remove or search in the archlinux.org database for the package.

### CODE REVIEW

**Comment by ManuelSchneid3r** [comment]
This is a cool idea. System updates would be cool too. Just food for thoughts for a future version.

File: Pacman.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                subtext = package["package"],
+                completion = "%s%s" % (__trigger__, package["name"]),
+                actions = [
+                    TermAction(
+                        text = "Install",
+                        commandline = ["pacman", "-S", package["name"]]
+                    ),
+                    TermAction(
+                        text = "Remove",
+                        commandline = ["pacman", "-R", package["name"]]
```
Comment (ManuelSchneid3r) [inline Pacman.py]: Please Use -Rs.

File: Pacman.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+""" pacman Package Manager Extension """
```
Comment (ManuelSchneid3r) [inline Pacman.py]: A more informative description would be cool , but its not a must.

**Comment by benedictdudel** [comment]
@ManuelSchneid3r How do I confirm the changes request? When viewing the change request I can just do a review comment.

**Comment by ManuelSchneid3r** [comment]
@benedictdudel actually I dont know, I never used it, sorry

**Comment by ManuelSchneid3r** [comment]
Okay this extension need some polishing. :D

**Comment by benedictdudel** [comment]
@ManuelSchneid3r Well, I dunno what I did tested there.. Is it even possible to run a `TermAction` as `sudo`? I always get `error: you cannot perform this operation unless you are root`.

**Comment by ManuelSchneid3r** [comment]
Sure but i fixed this already

**Pull request merged**